src/pages/test-report/components/serious-fault-badge/serious-fault-badge.scss

```diff
@@ -0,0 +1,20 @@
+serious-fault-badge {
+  div {
+    width: 22px;
+    height: 22px;
+    min-width: 22px;
+
+    .background {
+      background: map-get($colors-gds, 'gds-yellow');
+      color: #000000;
+      border-radius: 13px;
+
+      .label {
+        font-size: 15px;
+        font-weight: bold;
+        position: relative;
+        top: -4px;
```

Comment:
should really be vertically centred rather than, adjust via top.
use display flex, justify (on the parent element) and align-self on the child.

Reply:
It's the same as the DF badge :). Got a list of things from nad we need to tweak which include this. I was going to suggest sticking them in a new pr